server/routes/tenants/create_tenant.ts

```diff
@@ -0,0 +1,319 @@
+/*
+ * Copyright 2021-2024 Bitergia
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import path from 'path';
+import FormData from 'form-data';
+import StreamZip from 'node-stream-zip';
+import axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
+import { schema } from '@osd/config-schema';
+import { ImportResponse } from 'server/types';
+import { IBasePath, IRouter } from '../../../../../src/core/server';
+import { getRolesMapping } from './roles_mapping';
+
+export function registerTenantRoute(router: IRouter, basePath: IBasePath) {
```

Comment:
Shouldn't it be better to name this registerCreateTenantRoute? I guess each route is for one and only one API endpoint, right?

Reply:
Yes, that is a more accurate name. I've renamed it.